curriculum/challenges/english/25-front-end-development/quiz-css-backgrounds-and-borders/66ed8fd7f45ce3ece4053eb0.md

```diff
@@ -391,65 +407,81 @@ Which of these is NOT a valid border value?
 
 #### --text--
 
-How can you ensure that gradients used in web design are accessible?
+What is does the `radial-gradient` function do?
 
 #### --distractors--
 
-Use only two colors in the gradient
+It creates a background that radiates and transitions between multiple colors along a straight line.
 
 ---
 
-Use gradients only for decorative purposes
+It specifies whether the background image should scroll with the content or remain fixed in place.
 
 ---
 
-Avoid using gradients in the background
+It is used to style an element that was activated by the user.
 
 #### --answer--
 
-Ensure sufficient contrast between the gradient and any text.
+It creates an image that radiates from a particular point and gradually transitions between multiple colors.
 
 ### --question--
 
 #### --text--
 
-Which value for the `background-size` property will resize the background image to fit within the container while preserving the image's aspect ratio?
+Which of the following is NOT a valid pseudo-class?
 
 #### --distractors--
 
-`fill`
+`:focus`
 
 ---
 
-`stretch`
+`:visited`
 
 ---
 
-`repeat`
+`:link`
 
 #### --answer--
 
-`contain`
+`:before`
 
 ### --question--
 
 #### --text--
 
-How do you make an image border have rounded corners in CSS?
+Which of the following is the correct way to use the `background` shorthand property?
```

Comment:
```suggestion
Which of the following shows the `background` shorthand property being used properly?
```
Nit: the original implies there's one correct way, but it's just an example of one.